objectRecognitionSource/build.gradle

```diff
@@ -97,22 +101,22 @@ applicationDistribution.from("src/main/resources") {
 }
 
 // Downloads the large (100 MB+) video and graph files used for Yolo tests
-task addTestFiles {
-    // Download 'sampleVideo.mov' if it doesn't already exist
-    def sampleVideoUrl = new URL("https://www.dropbox.com/s/h5yd00qr8sfgdlt/sampleVideo.mov?dl=1")
-    def sampleVideoFile = file('src/test/resources/sampleVideo.mov')
-    if (!sampleVideoFile.exists()) {
-        sampleVideoUrl.withInputStream{ i -> sampleVideoFile.withOutputStream{ it << i }}
-    }
-            
-    // Download 'yolo.pb' if it doesn't already exist
-    def yoloGraphUrl = new URL("https://www.dropbox.com/s/lhnygcd6o6275jb/yolo.pb?dl=1")
-    def yoloGraphFile = file('src/test/resources/models/yolo.pb')
-    if (!yoloGraphFile.exists()) {
-        yoloGraphUrl.withInputStream{ i -> yoloGraphFile.withOutputStream{ it << i }}
-    }
-    
-}
+//task addTestFiles {
```

Comment:
Just checking that you intentionally left these commented lines here.

Reply:
yes, I did -- I wanted to leave a record of where the old stuff was for a little while.  We can evaporate it later.